lib/conjur/debify.rb

```diff
@@ -294,9 +299,16 @@ def copy_packages_from_container(container, package_name, dev_package_name)
 
       DebugMixin.debug_write "Built fpm image '#{image.id}' for project #{project_name}\n"
 
+      container_cmd_options = [ project_name, version ]
+
+      # Set the output file type if present
+      file_type = cmd_options[:output]
+      file_type ||= DEFAULT_FILE_TYPE
```

Comment:
this can change to `file_type = cmd_options[:output] || DEFAULT_FILE_TYPE`

Reply:
Done